ui/src/components/vault/VaultConfigure/VaultConfigure.jsx

```diff
@@ -101,8 +102,14 @@ function VaultConfigure({
   const [toLock, setToLock] = useState(
     AmountMath.makeEmpty(vaultCollateral.brand),
   );
+
   const [collateralPercent, setCollateralPercent] = useState(
-    vaultCollateral.initialMargin,
+    makeRatio(
+      vaultCollateral.liquidationMargin.numerator.value + 25n,
```

Comment:
I'm just auto-populating an initial value of liquidationMargin + 25%, since liquidationMargin + 0% doesn't make sense as everyone will need to change it. We could just not auto-populate the form but that requires more design thought.

Reply:
seems like a reasonable choice.

Though the denominator might be 100 or it might be 10,000. Does it make sense to multiply by 5/4 instead of adding 25?

Reply:
Changed it to add `denominator.value / 4` instead as discussed